### PR TITLE
Add DNS_INAPPROPRIATE_TTL_VALUES KB

### PR DESCRIPTION
**Pull request by elyousfi5**
KB for INAPPROPRIATE TTL VALUES
image

### CODE REVIEW

File: WEB_SERVICE/WEB/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/description.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Time-to-Live (TTL) values in DNS records define how long a DNS resolver should cache the record before querying the authoritative DNS server again. Incorrectly setting TTL values, whether too high or too low, can lead to inefficiencies in DNS propagation, which can impact website performance, DNS load balancing, and service failover.
+
+Setting the right TTL values is essential for balancing performance and DNS efficiency. Inappropriate TTL values can cause various issues, depending on whether they are set too high, too low, or not set at all. Each of these scenarios can have negative consequences:
```
Comment (Zakaria-Latif) [inline WEB_SERVICE/WEB/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/description.md:3]: highlight more explicitly how this affects DNS propagation, specifically in the context of changes to DNS records..

**Review by Zakaria-Latif — CHANGES_REQUESTED** [review]

File: WEB_SERVICE/WEB/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/meta.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "references": {
+    "Understanding DNS TTL": "https://convesio.com/knowledgebase/article/understanding-dns-time-to-live-a-comprehensive-guide/",
+    "DNS TTL Monitoring Guide": "https://www.catchpoint.com/dns-monitoring/dns-ttl",
+    "Best Practices for DNS TTL": "https://www.varonis.com/blog/dns-ttl"
+  },
+  "title": "DNS Inappropriate TTL Values",
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
+    "PCI_STANDARDS": [
```
Comment (ohachimOs) [inline WEB_SERVICE/WEB/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/meta.json:13]: Do these fit? as they are related to Payment

**Review by Zakaria-Latif — COMMENTED** [review]
I think this KB needs to be in the `WEB_SERVICE/DNS` directory

File: WEB_SERVICE/WEB/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/description.md (hunk 1 above)
Comment (PiranhaSa) [reply WEB_SERVICE/WEB/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/description.md:3]: it's mentioned here "- **High TTL values (e.g., greater than 86400 seconds)**: Reduce query traffic and enhance performance, but can delay important DNS updates (e.g., IP address changes), potentially causing downtime or outdated records being served.  "

**Review by PiranhaSa — APPROVED** [review]

File: WEB_SERVICE/WEB/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/meta.json (hunk 2 above)
Comment (elyousfi5) [reply WEB_SERVICE/WEB/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/meta.json:13]: fixed

**Review by Zakaria-Latif — APPROVED** [review]

File: WEB_SERVICE/DNS/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/meta.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "short_description": "Inappropriate TTL values in DNS records can cause performance issues and delays in DNS updates.",
+  "references": {
+    "Understanding DNS TTL": "https://convesio.com/knowledgebase/article/understanding-dns-time-to-live-a-comprehensive-guide/",
+    "DNS TTL Monitoring Guide": "https://www.catchpoint.com/dns-monitoring/dns-ttl",
+    "Best Practices for DNS TTL": "https://www.varonis.com/blog/dns-ttl"
+  },
+  "title": "DNS Inappropriate TTL Values",
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
```
Comment (nmasdoufi-ol) [inline WEB_SERVICE/DNS/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/meta.json:12]: Consider adding:

- CWE-400: Uncontrolled Resource Consumption
- CWE-611: Improper Restriction of XML External Entity (XXE) Reference

**Review by nmasdoufi-ol — APPROVED** [review]

**Review by ohachimOs — APPROVED** [review]

**Pull request merged**